### PR TITLE
Remove config yii-web, move aliases to level app.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

Merge firts: https://github.com/yiisoft/yii-web/pull/361

### CODE REVIEW

**Comment by terabytesoftw** [comment]
If the idea is correct I can change it, in the other packages.

File: yiisoft/middleware-dispatcher/config/web.php [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-
-declare(strict_types=1);
-
-use Yiisoft\Middleware\Dispatcher\MiddlewareFactory;
-use Yiisoft\Middleware\Dispatcher\MiddlewareFactoryInterface;
-use Yiisoft\Middleware\Dispatcher\MiddlewarePipelineInterface;
-use Yiisoft\Middleware\Dispatcher\MiddlewareStack;
-
-return [
-    MiddlewarePipelineInterface::class => MiddlewareStack::class,
```
Comment (samdark) [inline yiisoft/middleware-dispatcher/config/web.php:11]: Are you sure it should be removed?

**Review by samdark — COMMENTED** [review]
Overall 👍

File: yiisoft/middleware-dispatcher/config/web.php (hunk 1 above)
Comment (terabytesoftw) [reply yiisoft/middleware-dispatcher/config/web.php:11]: If I removed it because it is not in `config/yiisoft/packages`, I thought it was an error.

**Review by samdark — APPROVED** [review]

**Pull request merged**